torch/_export/non_strict_utils.py

```diff
@@ -618,6 +619,19 @@ class _NonStrictTorchFunctionHandler(torch.overrides.TorchFunctionMode):
     """
 
     def _override(self, func, args, kwargs):
+        if func is torch.distributed.all_reduce:
```

Comment:
do you want to make this more general? for all functional collectives?

Reply:
yeah, after this lands / doesn't get reverted I'll do others. Just want to feel this out first.